src/monitor/monitor.c

```diff
@@ -1895,6 +1779,9 @@ static void monitor_restart_service(struct mt_svc *svc)
     }
 }
 
+/* from nscd.c */
+errno_t sss_nscd_parse_conf(const char *conf_path);
```

Comment:
Shouldn't this declaration go in a header file?

Reply:
What for..? This function is used in a single place, i.e. not shared.